viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1220,6 +1223,34 @@ export class Cognite3DViewer {
     return this.intersectModels(offsetX, offsetY);
   }
 
+  /**
+   * Update near and far plane buffers by creating a combined bounding box from all objects in the scene.
+   * Note: this is automatically called when 3D models are added to or removed from the viewer.
+   */
```

Comment:
Say something about that this only should be necessary if you have custom objects whose bounds change.

Reply:
Also, I think this is a bit dodgy - when `automaticNearFarPlane` is false, this will have no effect. As a user I would expect that this _always_ sets near/far, regardless of the `automaticNearFarPlane`-flag.

Reply:
Agree, maybe we can rename the method again to something like `forceUpdateCameraNearAndFarBoundingBox`, so that users understand that it is not about updating actual near and far values but just some data for their calculations.